contents/tutorials/how-to-rename-events.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: How to rename events
+sidebar: Docs
+showTitle: true
+author:
+  - bijan-boustani
+date: 2024-10-22
+tags:
+  - actions
+  - events
+---
+
+**Estimated reading time:** 3 minutes ☕☕
+
+**tl;dr** Use [Actions](/docs/data/actions). Start capturing events with the new name, and then create an Action to group it with the existing event.
```

Comment:
Our current style is to lowercase these thigns (i.e dont treat them as proper nouns). Can you fix in the rest of the tutorial too please
```suggestion
**tl;dr** Use [Actions](/docs/data/actions). Start capturing events with the new name, and then create an action to group it with the existing event.
```